scripts/plot_dataframe.py

```diff
@@ -35,55 +32,113 @@
 logger = logging.getLogger(__name__)
 _CONF: Dict[str, Any] = {}
 
+
+# Update the global variable TO_DISPLAY to select which indicator you want to display
+TO_DISPLAY = {
+    'sma': False,  # On Row 1
+    'ema': True,  # On Row 1
+
+    'macd': True,  # On Row 3
+    'rsi': False,  # On Row 3
+    'fisher_rsi': False,  # On Row 3
+    'mfi': False,  # On Row 3
+    'slow': False,  # On Row 3
+    'fast': False  # On Row 3
+}
+
+
 def plot_analyzed_dataframe(args: Namespace) -> None:
     """
     Calls analyze() and plots the returned dataframe
     :return: None
     """
-    pair = args.pair.replace('-', '_')
+
+    # Load the configuration
+    config = setup_configuration(args)
+
+    # Set the pair to audit
+    pair = args.pair
+
+    # Set timerange to use
     timerange = Arguments.parse_timerange(args.timerange)
 
-    # Init strategy
+    # Load the strategy
     try:
-        analyze = Analyze({'strategy': args.strategy})
+        analyze = Analyze(config)
+        exchange.init(config)
     except AttributeError:
         logger.critical(
             'Impossible to load the strategy. Please check the file "user_data/strategies/%s.py"',
             args.strategy
         )
         exit()
 
-    tick_interval = analyze.strategy.ticker_interval
+    # Set the ticker to use
+    tick_interval = analyze.get_ticker_interval()
 
+    # Load pqir tickers
     tickers = {}
     if args.live:
         logger.info('Downloading pair.')
         # Init Bittrex to use public API
```

Comment:
Probably should remove the comment about Bittrex here

Reply:
Fixed